From 47c9c48ba2ad4abbc6c4932c32f74e4fdd8aa4ae Mon Sep 17 00:00:00 2001
From: Rob Evers <revers@redhat.com>
Date: Wed, 20 Aug 2014 18:08:56 -0400
Subject: [scsi] be2iscsi: Bump driver version

Message-id: <1408558136-21958-5-git-send-email-revers@redhat.com>
Patchwork-id: 88416
O-Subject: [RHEL7.1 e-stor PATCH 5/5] scsi: be2iscsi: Bump driver version
Bugzilla: 1130072
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: John Soni Jose <sony.john-n@emulex.com>

http://marc.info/?l=linux-scsi&m=140747402505708&w=2

Red Hat specific

Signed-off-by: John Soni Jose <sony.john-n@emulex.com>
Signed-off-by: Jayamohan Kallickal <jayamohan.kallickal@emulex.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/be2iscsi/be_main.h b/drivers/scsi/be2iscsi/be_main.h
index c7e7afb..25fae06 100644
--- a/drivers/scsi/be2iscsi/be_main.h
+++ b/drivers/scsi/be2iscsi/be_main.h
@@ -36,7 +36,7 @@
 #include <scsi/scsi_transport_iscsi.h>
 
 #define DRV_NAME  "be2iscsi"
-#define BUILD_STR  "10.2.273.0r"
+#define BUILD_STR  "10.4.74.0r"
 #define BE_NAME   "Emulex OneConnect" \
     "Open-iSCSI Driver version" BUILD_STR
 #define DRV_DESC  BE_NAME " " "Driver"
-- 
1.7.1